content/manuals/retired.md

```diff
@@ -0,0 +1,175 @@
+  - /registry/spec/auth/jwt/
+  - /registry/spec/auth/oauth/
+  - /registry/spec/auth/scope/
+  - /registry/spec/auth/token/
+  - /registry/spec/deprecated-schema-v1/
+  - /registry/spec/implementations/
+  - /registry/spec/json/
+  - /registry/spec/manifest-v2-1/
+  - /registry/spec/manifest-v2-2/
+  - /registry/spec/menu/
+  - /registry/storage-drivers/
+  - /registry/storage-drivers/azure/
+  - /registry/storage-drivers/filesystem/
+  - /registry/storage-drivers/gcs/
+  - /registry/storage-drivers/inmemory/
+  - /registry/storage-drivers/oss/
+  - /registry/storage-drivers/s3/
+  - /registry/storage-drivers/swift/
+---
+
+This document provides an overview of Docker features, products, and
+open-source projects that have been deprecated, retired, or transitioned.
+
+> [!NOTE]
+>
+> This page does not cover deprecated and removed Docker Engine features.
+> For a detailed list of deprecated Docker Engine features, refer to the
+> [Docker Engine Deprecated Features documentation](/manuals/engine/deprecated.md).
+
+## Products and features
+
+Support for these deprecated or retired features is no longer provided by
+Docker, Inc. The projects that have been transitioned to third parties continue
+to receive updates from their new maintainers.
+
+### Docker Machine
+
+Docker Machine was a tool for provisioning and managing Docker hosts across
+various platforms, including virtual machines and cloud providers. It is no
+longer maintained, and users are encouraged to use Docker Desktop or Docker
+Engine directly on supported platforms. Machine's approach to creating and
+configuring hosts has been superseded by more modern workflows that integrate
+more closely with Docker Desktop.
+
+### Docker Toolbox
+
+Docker Toolbox was used on older systems where Docker Desktop could not run. It
+bundled Docker Machine, Docker Engine, and Docker Compose into a single
+installer. Toolbox is no longer maintained and is effectively replaced by
+Docker Desktop on current systems. References to Docker Toolbox occasionally
+appear in older documentation or community tutorials, but it is not recommended
+for new installations.
+
+### Docker Cloud Integrations
+
+Docker previously offered integrations with multiple cloud providers to
+streamline container workflows. These integrations have been deprecated, and
+users should now rely on native cloud tools or third-party solutions to manage
+their workloads. The move toward platform-specific or universal orchestration
+tools reduced the need for specialized Docker Cloud integrations.
+
+### Docker Enterprise Edition
+
+Docker Enterprise Edition (EE) was Docker's commercial platform for deploying
+and managing large-scale container environments. It was acquired by Mirantis in
+2019, and users looking for enterprise-level functionality can now explore
+Mirantis Kubernetes Engine or other products offered by Mirantis. Much of the
+technology and features found in Docker EE have been absorbed into the Mirantis
+product line.
```

Comment:
Can we add a note here pointing users to the the Docker Business section for enterprise-level features?